adbms/src/test/scala/de/up/hpi/informationsystems/adbms/function/SequentialFunctionTest.scala

```diff
@@ -0,0 +1,78 @@
+package de.up.hpi.informationsystems.adbms.function
+
+import akka.actor.{Actor, ActorLogging, ActorRef, Terminated}
+import de.up.hpi.informationsystems.adbms.function.SequentialFunction.SequentialFunctionDef
+import de.up.hpi.informationsystems.adbms.protocols.RequestResponseProtocol
+import de.up.hpi.informationsystems.adbms.protocols.RequestResponseProtocol.{Request, Response, Success}
+import de.up.hpi.informationsystems.adbms.relation.Relation
+import org.scalatest.{Matchers, WordSpec}
+
+class SequentialFunctionTest extends WordSpec with Matchers{
+
+  "A SequentialFunction" when {
```

Comment:
You can take a look [here](https://doc.akka.io/docs/akka/2.5/testing.html) for testing actors